.github/workflows/documentation.yaml

```diff
@@ -28,7 +20,10 @@ jobs:
 
       - name: 👷 Generate zsd
         run: |
-          make doc
+          make doc-container
+          rc="$?"
+          sudo chown -R "$(id -u):$(id -g)" .
```

Comment:
Why is this needed?

Reply:
There's an explicit chown to the user running in the container in the command run by the container. This is resetting the perms afterwards, w/o it the PDF step fails for example.